src/pymmcore_widgets/_mda/_channel_table_widget.py

```diff
@@ -109,6 +109,19 @@ def __init__(
 
         group_layout.addWidget(wdg, 0, 1)
 
+        self._mmc.events.systemConfigurationLoaded.connect(self._clear)
```

Comment:
did you mean to connect `_clear` or `_on_sys_cfg_loaded`?    if the former, update the disconnect, if the latter, update here